app/views/contributions/activity_timeline.ctp

```diff
@@ -93,8 +93,25 @@ foreach ($stats as $stat) {
         echo $html->tag('td', $bar, array('class' => 'bar'));
         echo '</tr>';
 
+        $totalSentences += $numSentences;
     }
     echo '</table>';
+    
+    if( $month == date('m') && $year == date('Y')) {
+        $numberOfDays = date('d');
+    } else if (($year < date('Y')) || ($year == date('Y') && $month < date('m'))){
+        $numberOfDays = cal_days_in_month(CAL_GREGORIAN, $month, $year);
+    }
+    
+    if($numberOfDays > 0){
+        $dailyAverage = $totalSentences / $numberOfDays;
+        if($dailyAverage == 1){
+            echo '<div class="daily-average">Daily Average: ' . $dailyAverage . ' sentence</div>';         
+        } else {
```

Comment:
You can use `$html->div($className, $content)` instead of writing the HTML code.
Cf. http://book.cakephp.org/1.3/en/The-Manual/Core-Helpers/HTML.html#div

And you will need to use the `__()` function, or more precisely the `__n()` function, to output the `Daily average` string. This is needed to make the string translatable.
* Related doc: http://book.cakephp.org/1.3/en/The-Manual/Common-Tasks-With-CakePHP/Internationalization-Localization.html
* Example in the code: https://github.com/Tatoeba/tatoeba2/blob/dev/app/views/sentences_lists/show.ctp#L52